src/Bundles/Storage/PHP/Services/PHPLoader.php

```diff
@@ -31,6 +31,10 @@ public function loadTranslationSet(TranslationSet $set, string $delimiter): void
                 $locale->addTranslation($key, $value, '');
             }
 
+            // We start with three because the first three lines are the <?php tag and the return statement
+            $locale->setLineNumbers(
+                $this->getLineNumbers($arrayData, $delimiter, '', 3, true)
```

Comment:
im not quite sure if we can always say this
i think i might go with a "simple line reader" in that case
but i can do that later on , no worries

Reply:
I will have a look at it and try to implement it my self. Otherwise I will get back to you and we can implement it later :D

Reply:
ok sure
but i can also just merge it and see if it can be improved, because i would somehow love to merge it and prepare a release soon haha :)

Reply:
Yeah, feel free to merge. I can definitely add that afterwards 👍🏻